src/Google.Maps/Geocoding/GeocodeResponse.cs

```diff
@@ -20,7 +20,6 @@
 
 namespace Google.Maps.Geocoding
 {
-	[Serializable]

```

Comment:
It appears binary serialization was removed in .Net Core (looks like it's coming back in 2.0 though).

If Binary Serialization isn't being used then this attribute can be dropped as JSON serializers no longer need it.